Packs/CommonScripts/Scripts/StixCreator/StixCreator.py

```diff
@@ -307,6 +309,12 @@ def main():
 
                         kwargs['is_family'] = argToBoolean(xsoar_indicator.get('ismalwarefamily', 'False').lower())
 
+                    if indicator_type == 'report':
+                        kwargs['name'] = xsoar_indicator.get('name', '')
```

Comment:
```suggestion
                        kwargs['name'] = xsoar_indicator.get('value', '')
```
The name of a report is its value in XSOAR